ansible/roles/grafana/tasks/setup-nginx.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Install nginx
+  environment:
+    RUNLEVEL: 1
```

Comment:
IMHO, it deserves a comment like `# to avoid nginx being launched right after installation`